tools/src/main/java/org/apache/kafka/trogdor/workload/ConsumeBenchWorker.java

```diff
@@ -360,6 +360,7 @@ StatusData update() {
                 latSummary.percentiles().get(1).value(),
                 latSummary.percentiles().get(2).value());
             statusUpdater.updateConsumeStatus(consumer.clientId(), statusData);
+            statusUpdater.update();
```

Comment:
I think this call might not be needed, as we already have the `statusUpdater` scheduled to run `update()` once every minute and the `ConsumeStatusUpdater` scheduled to run once every minute.
I'm wondering if a minute is too long of a time frame and whether or not we should edit it

Reply:
Also, I know that this isn't from this PR but I just noticed it. On line 314, we have `WorkerUtils.abort(log, "ConsumeStatusUpdater", e, doneFuture);` whereas the string should be `StatusUpdater`. Is it possible we update that while we're at it?